lib/addons/p5.dom.js

```diff
@@ -61,9 +62,9 @@
    * // these are all valid calls to select()
    * var a = select('#moo');
    * var b = select('#blah', '#myContainer');
-   * var c = select('#foo', b);
+   * var c = b && select('#foo', b);
```

Comment:
I think I'd prefer adding `notest` to the example's classes as this sort of syntax is not the clearest for beginners, I'd have though. Open to this, though, depending upon other views

Reply:
yes i think i'd agree here..